getUser.php

```diff
@@ -269,6 +269,15 @@ function getSeasonName($seasonId) {
 		case 23:
 			$season_name = "Crystal Guard";
 		break;
+		case 24:
+			$season_name = "High Calibre";
+		break;
+		case 25:
+			$season_name = "Demon Veil";
+		break;
+		case 25:
+			$season_name = "Operation Vector Glare";
```

Comment:
Shouldn't `Operation Vector Glare` just be `Vector Glare` same way `Operation Health` is just `Health`

Reply:
Yeah i thought so too, but on Ubisofts official site. Operation Health and Operation Vector Glare are explicity written with the Prefix [https://www.ubisoft.com/en-gb/game/rainbow-six/siege/game-info/seasons](https://www.ubisoft.com/en-gb/game/rainbow-six/siege/game-info/seasons)